docs/sources/clients/promtail/stages/logfmt.md

```diff
@@ -0,0 +1,86 @@
+---
+title: logfmt
+---
+# `logfmt` stage
```

Comment:
As we add new content to the Loki documentation, we're trying to standardize what we include in the YAML front matter.  This suggestion adds 2 new fields to implement that.  I'm leaving out the weight field, as adding it only to this stage's file would mess up our (fine) alphabetized organization.

```suggestion
title: logfmt
menuTitle: logfmt
description: The logfmt parsing stage reads logfmt log lines and extracts the data into labels.
---
# `logfmt` stage
```